### PR TITLE
Make pull request on coala fix

### LINKED ISSUE #633: Automate integration of Coala patches
Currently -- we have coala pushing fixes to a [separate branch](https://github.com/fossasia/gci16.fossasia.org/tree/coala-fix). What would be REALLY cool, is if it created a pull request with the changes so we could merge them faster. 

There is a [script](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/scripts/coala_patch.sh), this script is **NOT** run for pull requests. 

Your task: have it make a pull request!

### PR DESCRIPTION
**Pull request by m1guelpf**
Fix #633

### CODE REVIEW

**Comment by robbyoconnor** [comment]
Before this is merged, amend your commit message please @m1guelpf

File: scripts/coala_patch.sh [hunk 1]
```diff
@@ -8,3 +8,4 @@ git commit --message "Coala auto-patch for Travis CI Build:$TRAVIS_BUILD_NUMBER
 git remote add origin-pages https://coala-autofix-bot:$COALA_FIXER@github.com/fossasia/gci16.fossasia.org.git
 git pull --rebase origin-pages coala-fix
 git push --quiet --set-upstream origin-pages coala-fix
+curl -H "Authorization: token $GITHUB_TOKEN" -d '{"title":"coala-fixes", "body":"Automated Coala fixes", "head":"coala-autofix", "base":"master"}' https://api.github.com/repos/fossasia/gci16.fossasia.org/pulls
```
Comment (robbyoconnor) [inline scripts/coala_patch.sh]: Make this more robust...check if things actually changed...

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Comment by robbyoconnor** [comment]
@m1guelpf, ping me on slack when you're ready. I will likely look at this later today when I wake up.

**Comment by m1guelpf** [comment]
@robbyoconnor Travis does include cURL. It appears to fail because of the token. I requested it in Slack to @Abhi2424shek

File: scripts/coala_patch.sh [hunk 2]
```diff
@@ -8,3 +8,5 @@ git commit --message "Coala auto-patch for Travis CI Build:$TRAVIS_BUILD_NUMBER
 git remote add origin-pages https://coala-autofix-bot:$COALA_FIXER@github.com/fossasia/gci16.fossasia.org.git
 git pull --rebase origin-pages coala-fix
 git push --quiet --set-upstream origin-pages coala-fix
+response = $(curl --write-out '%{http_code}' -output /dev/null -X POST--H "Authorization: token $GITHUB_TOKEN" --data '{"title":"coala-fixes", "body":"Automated Coala fixes", "head":"coala-autofix", "base":"master"}' https://api.github.com/repos/fossasia/gci16.fossasia.org/pulls)
+echo $response
```
Comment (robbyoconnor) [inline scripts/coala_patch.sh]: Lines 11 and 12 don't work. 

Just write out the returned JSON, no need to be fancy. The line below is all you need...take it exactly...I added the build number so we know where it came from.

`curl -X POST -H "Authorization: token $GITHUB_TOKEN" --data '{"title":"coala-fixes for build $TRAVIS_BUILD_NUMBER", "body":"Automated Coala fixes:$TRAVIS_BUILD_NUMBER", "base":"gh-pages", "head":"coala-fix"}' https://api.github.com/repos/fossasia/gci16.fossasia.org/pulls`

**Review by robbyoconnor — DISMISSED** [review]

**Comment by m1guelpf** [comment]
@robbyoconnor Done

**Comment by robbyoconnor** [comment]
@m1guelpf, 

Also you have the head and base wrong:

Pay attention to detail as to how it's set up -- it's in that script!

**Comment by robbyoconnor** [comment]
will look when I get home

**Comment by m1guelpf** [comment]
Fixed branch name and added token.

**Comment by robbyoconnor** [comment]
I'll merge it when I get home.

**Comment by m1guelpf** [comment]
The $TRAVIS_BUILD_NUMBER in the PR body is not working because you are using single ' and double " quotes. If we find a way to unescape variables on the JSON, maybe we could send the PR to $TRAVIS_BRANCH instead of master, as, for example, the fixes for this PR are being pushed to the gh-pages branch.

**Comment by m1guelpf** [comment]
@robbyoconnor It's fixed. You can merge.

**Comment by robbyoconnor** [comment]
@m1guelpf -- I'll merge and make a small fix on my side. Good work.

**Pull request merged**